fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -1697,6 +1702,19 @@ class FMCMainDisplay extends BaseAirliners {
         }
     }
 
+    updateMinimums() {
+        const ssm = this.shouldTransmitMinimums() ? Arinc429Word.SignStatusMatrix.NormalOperation : Arinc429Word.SignStatusMatrix.NoComputedData;
+
+        this.arincMDA(this.perfApprMDA ? this.perfApprMDA : 0, ssm, 17, 131072, 0);
+        this.arincDH(this.perfApprDH ? this.perfApprDH : 0, ssm, 16, 8192, 0);
```

Comment:
The SSMs should also take into account which minima is in use/entered for each label. Only zero or one of them should be NO at any time.